api_pagopa.yaml

```diff
@@ -0,0 +1,254 @@
+swagger: "2.0"
+info:
+  version: 1.0.0
+  title: Proxy PagoPA
+  description: Proxy for PagoPA payment services
+basePath: /api/v1
+schemes:
+  - https
+paths:
+  "/payment-requests/{rptId}":
+    x-swagger-router-controller: PaymentController
+    get:
+      operationId: getPaymentInfo
+      summary: Get Payment Info
+      description: Retrieve information about a payment
+      parameters:
+        - name: rptId
+          in: path
+          required: true
+          schema:
```

Comment:
schema not allowed here.

to use a schema for parameters you must create a "parameters" section in the swagger specs the use 
look here for an example: https://github.com/teamdigitale/digital-citizenship-functions/blob/master/api/public_api_v1.yaml#L282

Reply:
ref. https://swagger.io/docs/specification/2-0/describing-parameters/